lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -432,13 +432,12 @@ export abstract class PerformanceClient implements IPerformanceClient {
         if (
             !request ||
             !(
-                event.name in
                 [
-                    PerformanceEvents.AcquireTokenSilent,
-                    PerformanceEvents.SsoSilent,
-                    PerformanceEvents.AcquireTokenRedirect,
-                    PerformanceEvents.AcquireTokenPopup,
-                ]
+                    PerformanceEvents.AcquireTokenSilent.toString(),
```

Comment:
You can use `event.name in Object.values([PerformanceEvents.AcquireTokenSilent, ...])` to avoid converting each entry to string